samples/qa-drag-and-drop/src/views/MainPanel.module.less

```diff
@@ -0,0 +1,16 @@
+// MainPanel.module.less
+//
+
+.mainPanel {
+    .firstItem, .secondItem {
+        color: white;
+    }
+
+    .firstItem {
+        background-color: #d95000;
+    }
+
+    .secondItem {
+        background-color: #aba534;
+    }
+}
```

Comment:
let's also keep the same background-colors on focus.  what do you think?

Reply:
I kept the styling, but changed border-color on focus to match background-color when not focused.